.circleci/scripts/firefox-install.sh

```diff
@@ -4,7 +4,7 @@ set -e
 set -u
 set -o pipefail
 
-FIREFOX_VERSION='83.0'
```

Comment:
Are we sure we want to update this value?  Shouldn't we keep this value at our min supported version ?

Reply:
Yes this was intentional, our min supported version is 68 for FF.
I updated this to be the latest FF release as most users are on the latest version(s). 
I also needed to update the browser to be able to control the download directory of FF.
I've recently updated Chrome to the latest stable version #15015